services/frontend-service/src/v2-ui/App/App.test.tsx

```diff
@@ -0,0 +1,28 @@
+/*This file is part of kuberpult.
```

Comment:
looks fine, just one suggestion:
Can we move the old code into a subdirectory like `legacy-ui` and have the new v2 things in the normal `ui` directory?
This way it would make it easier in the future.

Reply:
Was originally planning to have this when switchover was made to have v2 be the default option. 
But yes, will update that